src/style.css

```
@@ -1,3 +1,81 @@
+html {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  width: 100%;
```

```suggestion
  font-family: "Roboto", sans-serif;
```
You can specify the font family here to get it inherited

---

still not fixed